README.md

```diff
@@ -1 +1,87 @@
-solution-pack-data-leakage-threat-response
\ No newline at end of file
+# Data Leakage Threat Response Solution Pack
+
+## Release Information
+
+- Solution Pack Version: 1.0.0
+- Minimum Compatible FortiSOAR™ Version: 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Overview
+
+### Introduction
+
+*Data Leakage Threat Response Solution Pack* is designed to provide a set of investigation and utility playbooks to respond to data leakage.
+
+Configure Cloud Access Security Broker(CASB) connector such as Symantec CloudSOC to capture events of policy violation like 'External File Sharing'. It creates an alert of type 'Data Leakage', and then triggers a response workflow.
+
+Refer to Simulation Scenarios - Data Leakage to experience the use case without CASB configuration
+
+This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage.
```

Comment:
This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage -> This is a repeated text. it is already mentioned above.

Reply:
fixed it. Please check